### ISSUE TITLE
Default field labels should be humanized by default

### ISSUE THREAD

**Issue by apaganobeleno** [gobuffalo/tags#13]
By default tags is setting labels based on struct field names, so for the following struct:

```go
type Car struct {
    Plate            string
    BrandName string
}
```

When using `<%= f.InputTag("BrandName", {placeholder:"Brand Name"}) %>`, it will generate:

```html
<div class="form-group">
    <label>BrandName</label>
    <input class=" form-control" id="invoice-Reference" name="BrandName" placeholder="Brand Name" type="text" value="">
</div>
```

My thinking is we should turn BrandName into `Brand Name` by using `inflect`'s Title Case unless a label option is provided to the inputTag.

@markbates Thoughts ?

**Comment by markbates** [comment]
Yes!

**Comment by apaganobeleno** [comment]
closed by #15
